### ISSUE TITLE
ssort reformats dummy classes (conflict with black 24)

### ISSUE THREAD

**Issue by slafs** [bwhmather/ssort#113]
In black v24 a few new rules become stable.
One of them is

> Dummy class and function implementations consisting only of ... are formatted more compactly ([#3796](https://github.com/psf/black/pull/3796))

Most of the dummy bodies adjustment black makes are fine with ssort,
but a dummy class isn't.

Consider this simple file:

```py
class MyException(Exception):
    ...
```

black v24 now re-formats it as:
```py
class MyException(Exception): ...
```

while `ssort` re-formats it back to the original "shape".

I think this is a bug on ssort side.

We have both `ssort` and `black` configured in our pre-commit setup
and now we're unable to use them together.

**Comment by bwhmather** [comment]
This certainly breaks our "if code is ordered correctly then it won't be changed rule".

**Comment by bwhmather** [comment]
Thank you for reporting.  Have published a fix in 0.12.4.
